tests/pw/utils/summaryReporter.ts

```
@@ -71,14 +79,31 @@ export default class summaryReport implements Reporter {
+                summary.passed_tests.push(test.title);
+                summary.tests.push(test.title);
+                break;
+            case 'unexpected':
+                if (!summary.failed_tests.includes(test.title)) {
+                    summary.failed_tests.push(test.title);
+                    summary.tests.push(test.title);
+                }
+                break;
+            case 'flaky':
+                summary.flaky_tests.push(test.title);
+                const index = summary.failed_tests.indexOf(test.title);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)